README.md

```diff
@@ -0,0 +1,27 @@
+# Redis-tries Module #
+
+Goal: Add and implement support for prefix tries in Redis
+
+Tries: A trie, also known as a digital tree, radix tree or prefix tree, is an efficient search tree where keys are strings. The name for the data structure comes from the word reTrieval. Using a trie, search complexities can be brought to O(n) where n is the length of the key. Each node of the Trie has a character, indication whether the node represents the end of a word, and multiple branches. Each branch represents a possible character for the path of the key. The root node does not contain a letter character.
+
+Redis: Redis is an open-source, in-memory remote database that offers versatile modules and support for common data structures like hashes, lists and bitmaps. It serves as a popular platform for projects and problem solving.
+
+We will be implementing Redis support for the prefix trie first with basic functionality of searching, inserting and deleting and will optimize further based on the needs of the other developing functionalities like autocomplete, spellcheck and text searching.
+
+## Setting Up ##
+
+First clone the respository if you haven't already anywhere on your local system.
```

Comment:
Because the introduction just focused on Redis, I was confused which repository you meant. Say something like "this repository" or "the redis-tries repository."